### DISCUSSION TITLE
Deploy on DIGITALOCEAN APPS

### DISCUSSION THREAD

**Discussion by capoia** [nuxt/nuxt#16948] [Questions]
Has anyone already deployed nuxt3 to digital oceans apps?
I am having problems.
Inside my middleware I import the pinia store and in the build it says that the store file (/store/Auth/adoes not exist.
I don't have these problems with yarn dev.

The generated log.
[message.txt](https://github.com/nuxt/framework/files/9480557/message.txt)
The directory folders.
![Screen Shot 2022-09-02 at 16 22 36](https://user-images.githubusercontent.com/21159928/188223228-a0f2ea7c-7439-4b2e-8e21-0551feea556f.png)
The middleware.
![image](https://user-images.githubusercontent.com/21159928/188223775-f9231b5d-2ebb-46f5-86ce-079128012bec.png)

**Comment by chz** [comment]
Do you checked with node 16.x version ?

**Reply by capoia** [reply]
> Do you checked with node 16.x version ?

same problem: 
[build.txt](https://github.com/nuxt/framework/files/9480700/build.txt)

**Comment by capoia** [comment]
i change @/ to ../ but still the error: 
<img width="821" alt="image" src="https://user-images.githubusercontent.com/21159928/188227388-0861546a-b16a-47d8-ba30-44f32f51f54d.png">

**Comment by capoia** [comment]
any helps?
